x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_management/classic.tsx

```diff
@@ -144,6 +144,9 @@ export function ClassicStreamDetailManagement({
   }
 
   if (!isValidManagementSubTab(tab) || tabs[tab] === undefined) {
+    if (isLoading) {
```

Comment:
would be good if we can do something like:

```ts
if (isValidTab) {
  return <Wrapper/>;
 }
 
 if (isLoading) {
   // or display a loader?
   return null;
 }
 // redirect to enrich tab
 return <RedirectTo../>
 ```
 
 a simpler change can be return isLoading ? null : <RedirectTo/>
 
 bit of a nit, but I think the nested `isLoading` doesn't help readability